source/features/index.js

```diff
@@ -16,6 +16,12 @@ export const features = {
 		label: 'Preserve unsent text in the Messages modal when it closes',
 		fn: require('./preserve-text-messages').default
 	},
+	expandUrlsRemoveUtms: {
```

Comment:
I think think removing UTMs should be in the ID or filename. It's just an implementation detail.

```suggestion
	expandUrls: {
```